src/app/dynamic-forms/enums/tiptap-mark-type.enum.ts

```diff
@@ -92,7 +92,13 @@ export enum TiptapMarkType {
    */
   ListUnordered = "BULLET_LIST",
   /**
-   * Automatically inserts a <p> tag when doing a soft line breat
+   * Represents a <ul data-type="taskList">
```

Comment:
I would rephrase this to `Represents a <ul> tag with the data-type attribute set to "task-list"` or something like that